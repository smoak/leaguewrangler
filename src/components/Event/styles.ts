import { createStyles } from '@material-ui/core/styles';

export default (): ReturnType<typeof createStyles> =>
  createStyles({
    card: {
      width: '100%',
    },
  });
