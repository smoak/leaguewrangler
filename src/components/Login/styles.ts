import { Theme } from '@mui/material';

import createStyles from '@mui/styles/createStyles';

const styles = (theme: Theme): ReturnType<typeof createStyles> =>
  createStyles({
    main: {
      width: 'auto',
      display: 'block', // Fix IE 11 issue.
      marginLeft: theme.spacing(3),
      marginRight: theme.spacing(3),
      [theme.breakpoints.up('sm')]: {
        width: 552,
        marginLeft: 'auto',
        marginRight: 'auto',
      },
    },
    paper: {
      marginTop: theme.spacing(8),
      display: 'flex',
      flexDirection: 'column',
      alignItems: 'center',
      padding: `${theme.spacing(2)} ${theme.spacing(3)} ${theme.spacing(3)}`,
    },
    avatar: {
      margin: theme.spacing(),
      backgroundColor: theme.palette.secondary.main,
    },
    form: {
      width: '100%', // Fix IE 11 issue.
      marginTop: theme.spacing(),
    },
    submit: {
      marginTop: theme.spacing(3),
    },
  });

export default styles;
