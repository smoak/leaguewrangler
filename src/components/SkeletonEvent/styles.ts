import { createStyles } from '@material-ui/core/styles';

export default (): ReturnType<typeof createStyles> =>
  createStyles({
    card: {
      maxWidth: 354,
      width: '100%',
    },
  });
