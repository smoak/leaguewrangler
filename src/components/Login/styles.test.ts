import { Theme, createMuiTheme } from '@material-ui/core/styles';

import styles from './styles';

describe('styles', () => {
  let theme: Theme;

  beforeEach(() => {
    theme = createMuiTheme({
      typography: {
        useNextVariants: true,
      },
    });
  });

  it('generates styles correctly', () => {
    expect(styles(theme)).toMatchSnapshot();
  });
});
