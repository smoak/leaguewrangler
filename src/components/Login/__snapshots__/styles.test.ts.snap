// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`styles generates styles correctly 1`] = `
Object {
  "avatar": Object {
    "backgroundColor": "#f50057",
    "margin": 8,
  },
  "form": Object {
    "marginTop": 8,
    "width": "100%",
  },
  "main": Object {
    "@media (min-width:448px)": Object {
      "marginLeft": "auto",
      "marginRight": "auto",
      "width": 400,
    },
    "display": "block",
    "marginLeft": 24,
    "marginRight": 24,
    "width": "auto",
  },
  "paper": Object {
    "alignItems": "center",
    "display": "flex",
    "flexDirection": "column",
    "marginTop": 64,
    "padding": "16px 24px 24px",
  },
  "submit": Object {
    "marginTop": 24,
  },
}
`;
